lib/Wikia/src/Service/User/PreferencePersistence.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Wikia\Service\User;
+
+interface PreferencePersistence {
+
+	/**
+	 * Save the users preferences.
+	 *
+	 * @param int $userId
+	 * @param array [\Wikia\Domain\User\Preference, ... ]
```

Comment:
This can be properly annotated with `Preference[]` to indicate an array of `Preference` objects